pom.xml

```diff
@@ -163,8 +163,15 @@
     <build>
         <resources>
             <resource>
-                <filtering>false</filtering>
                 <directory>rules</directory>
+                <filtering>false</filtering>
+            </resource>
+            <resource>
+                <directory>rules-reviewed/jboss-eap/websphere</directory>
```

Comment:
I don't think this belongs in the commit? Alternatively, maybe it could include all of rules-reviewed instead?